### ISSUE TITLE
[gcdserver] unit tests need to be run automatically (Trac #2168)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2160]
They should be turned on if the environment contains zmq and pymongo.  Alternatively (probably better), just simulate a mongo database in memory.

Migrated from https://code.icecube.wisc.edu/ticket/2168
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:15:23", 
    "description": "They should be turned on if the environment contains zmq and pymongo.  Alternatively (probably better), just simulate a mongo database in memory.", 
    "reporter": "jbraun", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067323910946", 
    "component": "combo reconstruction", 
    "summary": "[gcdserver] unit tests need to be run automatically", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2018-06-27T15:24:52", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20181001T21:59:20: olivas changed summary from "gcdserver unit tests need to be run automatically" to "[gcdserver] unit tests need to be run automatically"

**Comment by icecube-trac** [comment]
Trac update at 20181008T22:06:41: olivas changed milestone from "Autumnal Equinox 2018" to "Winter Solstice 2018"

**Comment by icecube-trac** [comment]
Trac update at 20181210T21:05:22: 
* jbraun changed resolution from "" to "fixed"
* jbraun commented:

> In changeset 167461:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="167461"
> Run unit tests automatically.  Closes #2160.
> ```


* jbraun changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T04:03:11: nwhitehorn commented:

> In changeset 170608:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170608"
> Run unit tests automatically.  Closes #2160.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:15:23: 
* olivas commented:

> Milestone Winter Solstice 2018 deleted


* olivas changed milestone from "Winter Solstice 2018" to ""
